src/stratis_cli/_error_reporting.py

```diff
@@ -151,6 +151,9 @@ def _interpret_errors(errors):
             )
             return fmt_str % error
 
+        if isinstance(error, BrokenPipeError):  # pragma: no cover
```

Comment:
Please explain why this branch, specifically, is not tested. The way to do that might be to think about how you would add a test to the CI to test this branch, i.e., how you would reproduce the circumstances that would cause the broken pipe. Is there a reason why trying to put a test in the CI for this would be too much work or possibly self-defeating?